llama_hub/sec_filings/README.md

```diff
@@ -13,78 +13,27 @@ python install -r requirements.txt
 The SEC Downloader expects 5 attributes
 
 * tickers: It is a list of valid tickers
-* amount: Number of documents that you want to download
```

Comment:
As mentioned above, the previous implementation was breaking, and the amount parameter is a bit ambiguous. In my conversation, users would like to pull the documents for a given year or a list years, not number of filings. Hence, the year parameter serves better.

Reply:
Ok sounds good. In general we are trying to minimize the number of breaking changes, it's not good to switch user-facing params around because that breaks existing implementations.

If the previous implementation doesn't work at all then sure we can remove (and log a warning to the user that it no longer works). If it still does then I vote we leave in the parameter for backwards compat

Reply:
Yes, understood
In the latest commit, I have added the amount deprecating warning. Please do suggest, if I need to make other changes.